cmd/minishift/cmd/delete.go

```diff
@@ -36,6 +37,10 @@ var deleteCmd = &cobra.Command{
 }
 
 func runDelete(cmd *cobra.Command, args []string) {
+	instancedata.Data.ReadInstanceData()
+
+	fmt.Println("ocPath in delete ", instancedata.Data.OcPath)
```

Comment:
@budhrg we can remove it I think this is just as part of test out if we can read easily in the `delete`.